ee/tabby-webserver/src/service/email.rs

```diff
@@ -20,7 +20,7 @@ use crate::schema::{
     setting::SettingService,
 };
 
-struct EmailServiceImpl {
+pub struct EmailServiceImpl {
```

Comment:
no need for this change?

Reply:
Needed for a test in the auth service. To construct an auth service, a mail service is now needed, so the struct must be visible there.

Reply:
please see the comments below: https://github.com/TabbyML/tabby/pull/1424#discussion_r1486815204